sqlmesh/core/test/definition.py

```diff
@@ -101,18 +101,30 @@ def shortDescription(self) -> t.Optional[str]:
     def setUp(self) -> None:
         """Load all input tables"""
         for table_name, values in self.body.get("inputs", {}).items():
+            all_types_are_known = False
             columns_to_types: t.Dict[str, exp.DataType] = {}
             if table_name in self.models:
                 model = self.models[table_name]
-                columns_to_types = model.columns_to_types if model.annotated else {}
+                columns_to_types = {
```

Comment:
Maybe clarify that this is `known` columns_to_types.

Reply:
Done